MovieCave - MVVM/MovieCave/MovieCave/Custom UI/CastAndTrailerDetailsView/GenreReusable/GenreView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GenreLabel.swift
+//  MovieCave
+//
+//  Created by Admin on 29.12.23.
+//
+
+import UIKit
+
+class GenreView: UIView {
+    
+    //MARK: - Properties
+    private let label: UILabel
+    
+    var text: String? {
+        get {
+            return label.text
+        }
+        set {
+            label.text = newValue
+            updateViewSize()
+        }
+    }
+    
+    //MARK: - Initialization
+    init(text: String?) {
```

Comment:
In some custom views you have used setup.... protocol function but in this one you have a custom init. Both approaches are fine but currently it's not consistent. Make it consistent